app/views/playlists/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>Playlists#index</h1>
 <p>Find me in app/views/playlists/index.html.erb</p>
 <% playlists.each do |playlist| %>
-  <%= playlist.name %>
+  <%= link_to playlist.name, user_playlist_path(current_user, playlist) %>
```

Comment:
nice iterative approach from showing just the name to linking to something useful for the user